core/src/OpenMP/Kokkos_OpenMP_UniqueToken.hpp

```diff
@@ -105,7 +105,8 @@ class UniqueToken<OpenMP, UniqueTokenScope::Global> {
   /// \brief upper bound for acquired values, i.e. 0 <= value < size()
   KOKKOS_INLINE_FUNCTION
   int size() const noexcept {
-    KOKKOS_IF_ON_HOST((return Kokkos::Impl::g_openmp_hardware_max_threads;))
+    KOKKOS_IF_ON_HOST(
+        (return Kokkos::Impl::OpenMPInternal::max_hardware_threads();))
```

Comment:
Note that this now would involve a function call.
It looks like this is fine as far as `ScatterView` is concerned (not used in the hot path as far as I can tell) but we probably should discuss it a bit.